api/controllers/countries.js

```diff
@@ -112,7 +112,7 @@ function getCountrySpecies(req, res) {
     INNER JOIN species_country sc on sc.species_id = s.species_id
     INNER JOIN countries c on c.country_id = sc.country_id AND
       c.iso3 = '${req.params.iso}'
-    INNER JOIN populations_iba p on p.species_main_id = s.species_id
+    INNER JOIN populations p on p.species_main_id = s.species_id
```

Comment:
As the information in this populations table is actually related to IBAs, maybe keeping IBA would make sense.